### PR TITLE
Missing publisher tests.

### PR DESCRIPTION
**Pull request by matek2305**

### CODE REVIEW

File: src/test/groovy/com/hltech/judged/contracts/publisher/PublisherTest.groovy [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+                    "http://localhost:${wireMockRule.port()}",
+                    [],
+                    [],
+                    [:]
+            )
+        expect:
+            publisher.publish(properties)
+    }
+
+    def "should publish swagger contract when rest capabilities is provided"() {
```
Comment (Felipe444) [inline src/test/groovy/com/hltech/judged/contracts/publisher/PublisherTest.groovy]: are

File: src/test/groovy/com/hltech/judged/contracts/publisher/PublisherTest.groovy [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                    ['rest'],
+                    [],
+                    [swaggerLocation:'src/test/resources/swagger']
+            )
+
+        expect:
+            publisher.publish(properties)
+    }
+
+    def "should publish pacts when rest expectations is provided"() {
```
Comment (Felipe444) [inline src/test/groovy/com/hltech/judged/contracts/publisher/PublisherTest.groovy]: are

File: src/test/resources/vaunt/information-service.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
+{
+  "name": "information-service",
+  "capabilities": {
+    "contracts": [
+      {
+        "destinationType": "QUEUE",
+        "destinationName": "request_for_information_queue",
+        "message": {
+          "type": "object",
+          "id": "urn:jsonschema:com:hltech:vaunt:generator:domain:representation:RepresentationWriterSpec:RequestMessage",
```
Comment (Felipe444) [inline src/test/resources/vaunt/information-service.json]: in id Vaunt puts only actual name of the class representing the message, in this case: RequestMessage

File: src/test/resources/vaunt/information-service.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            }
+          }
+        }
+      },
+      {
+        "destinationType": "TOPIC",
+        "destinationName": "something_changed_topic",
+        "message": {
+          "type": "object",
+          "id": "urn:jsonschema:com:hltech:vaunt:generator:domain:representation:RepresentationWriterSpec:ChangedEvent",
```
Comment (Felipe444) [inline src/test/resources/vaunt/information-service.json]: in id Vaunt puts only actual name of the class representing the message, in this case: ChangedEvent

File: src/test/resources/vaunt/information-service.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  },
+  "expectations": {
+    "providerNameToContracts": {
+      "audit-service": [
+        {
+          "destinationType": "QUEUE",
+          "destinationName": "audit_queue",
+          "message": {
+            "type": "object",
+            "id": "urn:jsonschema:com:hltech:vaunt:generator:domain:representation:RepresentationWriterSpec:AuditMessage",
```
Comment (Felipe444) [inline src/test/resources/vaunt/information-service.json]: in id Vaunt puts only actual name of the class representing the message, in this case: AuditMessage

File: src/test/resources/vaunt/information-service.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+          }
+        }
+      ],
+      "remote-client": [
+        {
+          "destinationType": "QUEUE",
+          "destinationName": "reject_information_queue",
+          "message": {
+            "type": "object",
+            "id": "urn:jsonschema:com:hltech:vaunt:generator:domain:representation:RepresentationWriterSpec:RejectMessage",
```
Comment (Felipe444) [inline src/test/resources/vaunt/information-service.json]: in id Vaunt puts only actual name of the class representing the message, in this case: RejectMessage

File: src/test/resources/vaunt/information-service.json [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+              }
+            }
+          }
+        },
+        {
+          "destinationType": "QUEUE",
+          "destinationName": "accept_information_queue",
+          "message": {
+            "type": "object",
+            "id": "urn:jsonschema:com:hltech:vaunt:generator:domain:representation:RepresentationWriterSpec:AcceptMessage",
```
Comment (Felipe444) [inline src/test/resources/vaunt/information-service.json]: in id Vaunt puts only actual name of the class representing the message, in this case: AcceptMessage

File: src/test/resources/vaunt/information-service.json [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+              "id": {
+                "type": "integer"
+              }
+            }
+          }
+        }
+      ]
+    }
+  }
+}
```
Comment (Felipe444) [inline src/test/resources/vaunt/information-service.json]: no newline @EOF

File: src/test/resources/swagger/swagger.json [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+          "type": "integer",
+          "format": "int32"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    }
+  }
+}
```
Comment (Felipe444) [inline src/test/resources/swagger/swagger.json]: no newline @eof

File: src/test/resources/pact/sample-pact.json [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+        "body": {
+          "error": "Argh!!!"
+        }
+      }
+    }
+  ],
+  "metadata": {
+    "pactSpecificationVersion": "1.0.0"
+  }
+}
```
Comment (Felipe444) [inline src/test/resources/pact/sample-pact.json]: no newline @eof

File: src/test/resources/pact/another-pact.json [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+            "name": "Event Two"
+          }
+        ]
+      }
+    }
+  ],
+  "metadata": {
+    "pactSpecificationVersion": "2.0.0"
+  }
+}
```
Comment (Felipe444) [inline src/test/resources/pact/another-pact.json]: no newline @eof

**Review by Felipe444 — CHANGES_REQUESTED** [review]

**Comment by Felipe444** [comment]
Minor changes suggested.
Please remove old judge-d-publisher when new one is introduced wherever necessary.

File: src/test/groovy/com/hltech/judged/contracts/publisher/PublisherTest.groovy (hunk 1 above)
Comment (matek2305) [reply src/test/groovy/com/hltech/judged/contracts/publisher/PublisherTest.groovy]: fixed

File: src/test/groovy/com/hltech/judged/contracts/publisher/PublisherTest.groovy (hunk 2 above)
Comment (matek2305) [reply src/test/groovy/com/hltech/judged/contracts/publisher/PublisherTest.groovy]: fixed

File: src/test/resources/vaunt/information-service.json (hunk 3 above)
Comment (matek2305) [reply src/test/resources/vaunt/information-service.json]: fixed

File: src/test/resources/vaunt/information-service.json (hunk 4 above)
Comment (matek2305) [reply src/test/resources/vaunt/information-service.json]: fixed

File: src/test/resources/vaunt/information-service.json (hunk 5 above)
Comment (matek2305) [reply src/test/resources/vaunt/information-service.json]: fixed

File: src/test/resources/vaunt/information-service.json (hunk 6 above)
Comment (matek2305) [reply src/test/resources/vaunt/information-service.json]: fixed

File: src/test/resources/vaunt/information-service.json (hunk 7 above)
Comment (matek2305) [reply src/test/resources/vaunt/information-service.json]: fixed

File: src/test/resources/vaunt/information-service.json (hunk 8 above)
Comment (matek2305) [reply src/test/resources/vaunt/information-service.json]: added

File: src/test/resources/swagger/swagger.json (hunk 9 above)
Comment (matek2305) [reply src/test/resources/swagger/swagger.json]: added

File: src/test/resources/pact/sample-pact.json (hunk 10 above)
Comment (matek2305) [reply src/test/resources/pact/sample-pact.json]: added

File: src/test/resources/pact/another-pact.json (hunk 11 above)
Comment (matek2305) [reply src/test/resources/pact/another-pact.json]: added

**Review by Felipe444 — APPROVED** [review]

**Pull request merged**